core/block_svg.ts

```diff
@@ -126,7 +126,10 @@ export class BlockSvg
   style: BlockStyle;
   /** @internal */
   pathObject: IPathObject;
-  override rendered = false;
+
+  /** Is this block a BlockSVG? */
+  override readonly rendered = true;
```

Comment:
If the only purpose of this is to detect whether it's a `BlockSvg` I would rather just remove it. less chance of it being used mistakenly. an instanceof check for `BlockSvg` would be preferred.